toolkit/tools/pkg/imagecustomizerlib/shrinkfilesystems.go

```diff
@@ -57,8 +67,18 @@ func shrinkFilesystems(imageLoopDevice string, verityHashPartition *imagecustomi
 
 		logger.Log.Infof("Shrinking partition (%s)", partitionLoopDevice)
 
+		startSector, foundStartSector := startSectors[partitionLoopDevice]
+		if !foundStartSector {
+			return fmt.Errorf("failed to find start sector for partition (%s)", partitionLoopDevice)
```

Comment:
the errors from getStartSectors() don't seem to bubble up - is that intentional?

Reply:
I don't understand the question. The `err` returned by `getStartSectors()` is checked.

`getStartSectors()` is now returning a mapping from dev-path to start sector. The check on this line is ensuring the partition we are interested in, is actually contained within the map. This check is probably very unlikely to fail. But I like to be as robust as possible.

Reply:
Ah never mind got it, I see it on line 38. Ignore